Python/bytecodes.c

```diff
@@ -813,6 +813,13 @@ dummy_func(
         };
 
         inst(SEND, (unused/1, receiver, v -- receiver, retval)) {
+            if ((frame->owner == FRAME_OWNED_BY_GENERATOR) &&
```

Comment:
It would be better to put this after the `#if ENABLE_SPECIALIZATION` block, I think, since a failed specialization attempt jumps back to the start of the instruction. So anything that happens before the specialization attempt risks happening twice for a single execution of the opcode.

In this particular case it shouldn't ever actually happen twice, since a failed specialization attempt should mean the receiver is the wrong type, in which case we should not enter this block anyway. But that's a bit implicit across two different files; it would be safer to just do it later (and it avoids checking the conditions twice.)